build_tools/third_party/cuda/CMakeLists.txt

```diff
@@ -0,0 +1,78 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+set(TARGET_DIR ${CMAKE_CURRENT_BINARY_DIR})
+set(DOWNLOAD_SCRIPT_URL "https://raw.githubusercontent.com/NVIDIA/build-system-archive-import-examples/main/parse_redist.py")
```

Comment:
Should we pin this to a commit? I know it's Nvidia itself, but still might want to avoid supply chain attacks and mysterious failures that aren't deterministics with IREE commit.

Reply:
Pinned to the current commit. I will say the repo only has 5 commit: https://github.com/NVIDIA/build-system-archive-import-examples/commits/main

Doesn't look like its actively updated at this point.